apis/core.oam.dev/v1beta1/resourcetracker_types.go

```diff
@@ -81,6 +81,8 @@ type ManagedResource struct {
 	Data *runtime.RawExtension `json:"raw,omitempty"`
 	// Deleted marks the resource to be deleted
 	Deleted bool `json:"deleted,omitempty"`
+	// DependsOn records the component that was dependent
+	DependsOn []string `json:"dependsOn,omitempty"`
```

Comment:
```suggestion
	// DependsOn records the component name that was dependent
	DependsOnComponents []string `json:"dependsOn,omitempty"`
```

Reply:
what if the dependency is recorded as inputs/outputs? \cc @FogDong

Reply:
We can record both depends on and inputs/outputs in this field.

Reply:
> We can record both depends on and inputs/outputs in this field.

yes, the  inputs/outputs can be recorded in this field